src/main/java/frc/robot/Commands/DropPlacer.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.Commands;
+
+import com.spikes2212.command.genericsubsystem.commands.MoveGenericSubsystem;
+import com.spikes2212.dashboard.RootNamespace;
+import edu.wpi.first.wpilibj.Timer;
+import frc.robot.Subsystems.ClimberPlacer;
+
+import java.util.function.Supplier;
+
+public class DropPlacer extends MoveGenericSubsystem {
+
+    private final ClimberPlacer placer;
+
+    private static final RootNamespace rootNamespace = new RootNamespace("drop placers");
+    public static final Supplier<Double> initialWaitTime = rootNamespace.addConstantDouble("initial wait time", 0);
+
+    private double startTime;
+
+    public DropPlacer(ClimberPlacer placer) {
+        super(placer, placer.getDropSpeed());
+        this.placer = placer;
+    }
+
+    private boolean initialTimePassed() {
+        return Timer.getFPGATimestamp() - startTime > initialWaitTime.get();
+    }
+
+    @Override
+    public void initialize() {
+        startTime = Timer.getFPGATimestamp();
+        super.initialize();
+    }
+
+    @Override
+    public boolean isFinished() {
+        return super.isFinished() && placer.isStalling() && initialTimePassed();
```

Comment:
if one of the limits of the placer is pushed (the placer cant move), you should stop immediately

Reply:
return !(frontLimit.get() && speed < 0) && !(backLimit.get() && speed > 0); is in the canMove() of the climber placer. This should solve the problem.

Reply:
so put || instead of &&?

Reply:
no